packages/AI/VectorsPinecone/src/models/PineconeDatabase.ts

```diff
@@ -184,15 +184,8 @@ export class PineconeDatabase extends VectorDBBase {
                 }
 
                 let duplicate: PotentialDuplicate = new PotentialDuplicate();
+                duplicate.LoadFromList(metadata.PrimaryKeys);
```

Comment:
@JS-BC  shouldn't this be metadata.CompositeKey?

Reply:
The KeyValuePairs are saved to the Pinecone db under the name PrimaryKeys. It should be renamed to KeyValuePair so that its more clear